demo/text_matching/text_matching.py

```diff
@@ -0,0 +1,97 @@
+# coding:utf-8
```

Comment:
I don't thinks we need the coding utf-8 line.
Since PaddleHub should use English comments.

Reply:
Done